cluster/service/cluster_service.go

```diff
@@ -217,13 +217,20 @@ func ValidateURL(urlStr *string) error {
 
 // InitializeClusterWatcher initializes a file watcher for the cluster config file
 // When the file is updated the configuration synchronously reload the cluster configuration
-func (c clusterService) InitializeClusterWatcher() (func() error, error) {
+func (c clusterService) InitializeClusterWatcher() (func() error, chan bool, error) {
```

Comment:
Do you really still need that done channel in the tests after fixing the parallel test execution?

Reply:
right, it's not needed anymore, so I reverted that part.

Reply:
done in d78a591